case-studies/DLP.md

```diff
@@ -0,0 +1,60 @@
+
+Google Ventures' (GV) design sprint is perhaps the most popular published design sprint structure and likely the basis of many others. Since its publishing, there have been many adaptations and additions to exercises for particular situations. Although the stock GV sprint was preferred for MBL, Founders and Coders have since used adapted sprints tailored closer to the needs of their specific participants and problem areas.
+
+---
+
+## Structure
+
+### Discovery
+* Presenting the challenge
+* Mapping
+* Ask the experts
+* How Might We
+* Theming How Might We Questions
+* Dot voting
+* Placing How Might We questions on the map
+* Choosing a target
+
+### Definition
+* Lightning demos
+* 3 part Sketching
+    * Notes & Ideas
+    * Crazy 8s
+    * Solution Sketch
+* Heatmap
+* Dot voting
+* Storyboarding/User Flow
+
+### Prototyping
+* Create a prototype (figma)
+* Usability testing
+* Usability testing synthesis
+* Usability testing insights
+
+---
+
+## Reflections
+
+* Initially, workshops were planned to be a full day in length as would be normal for a face-to-face sprint. After a test session, a full day of remote sprint was discovered to be too long, and the programme was restructured to take place over half day workshops. While the length of sprint exercises is malleable to the length of time available, we found that many could be cut significantly while still being productive.
+* The three problem areas we began with focused on youth services issues, but were otherwise very broad. This created a wide range of areas that could be targeted for a digital solution and it became difficult to facilitate a decision with such a range of voices in the room. The GV design sprint recommends appointing a "Decider" who has final say. To keep things democratic this role was avoided for MBL, but in hindsight it may have been useful and is recommended if there is someone on the sprint team who would be appropriate for the role. In lieu of a Decider, dot voting is a good way to make choices. It may be wise with a smaller sprint team to allot more dot votes, otherwise it can be hard to identify a favourite amongst several solution options.
+* Storyboarding is the last exercise in the sprint as suggested by GV and involves drawing frames of the solution step-by-step as the user would see them. As this is a group activity and time consuming it was too difficult to replicate effectively during MBL using remote, online tools. Instead we used a 'user flow' exercise where the equivalent frame is described on a note in text. The frames could show different user journeys through the solution via arrows from note to note.
```

Comment:
Could we add a sentence on the effectiveness of User Flows? something along the lines of "This turned out to be a better approach for this context because..."